gdal/frmts/mrf/marfa.h

```diff
@@ -657,12 +660,14 @@ class JPEG_Codec {
     const ILImage img;
 
     // JPEG specific flags
-    bool sameres;
-    bool rgb;
-    bool optimize;
+    bool sameres;  // No color space subsample
+    bool rgb;      // No conversion to YCbCr
+    bool optimize; // Optimize Huffman tables
+    bool noJXL;    // Write JFIF only
```

Comment:
coud use instead a bool JXL variable ? (and initialize it here using C++11 inline initialization). Double-negations are hard to comprehend.

Reply:
I could, but all these default to false, and I want the brunsli to be the on by default when compiled in. iIt is easier to describe as "set if you want plain JFIF" than "unset if you don't want JXL", which would be a double negation in the documentation instead.